tern_api/__init__.py

```diff
@@ -1,7 +1,16 @@
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 #
 # Copyright (c) 2022 VMware, Inc. All Rights Reserved.
 # SPDX-License-Identifier: BSD-2-Clause
+import os
+
 from flask import Flask
+from flask_executor import Executor
+
+tern_app = Flask(__name__)
+tern_app.config["TERN_API_CACHE_DIR"] = os.getenv("TERN_API_CACHE_DIR")
+tern_app.config["TERN_DEFAULT_REGISTRY"] = os.getenv("TERN_DEFAULT_REGISTRY")
 
```

Comment:
The cache directory can be retrieved from tern using `tern.utils.rootfs.get_working_directory()`. You may also want to add a note in the README about setting the TERN_DEFAULT_REGISTRY value when not working with the docker container.